arch/riscv/Kconfig

```diff
@@ -190,7 +238,14 @@ config MAIN_STACK_SIZE
 	default 4096 if 64BIT
 	default 2048 if PMP_STACK_GUARD
 
+config IDLE_STACK_SIZE
+	default 4096 if 64BIT
+
+config PRIVILEGED_STACK_SIZE
+	default 4096 if 64BIT
+
 config TEST_EXTRA_STACK_SIZE
+	default 4096 if 64BIT
```

Comment:
Is this change related to your platform only? If that's the case, please put it in your SoC's `Kconfig.defconfig`.

Reply:
My experience when running some of the tests and sample applications is that the stack sizes are insufficient for Risc-V 64bit especially when compiling for debugging. Even more so when FPU support is added in! There are a number of apps in the Zephyr repository that have fixed stack sizes that I've had to tweak and where possible I've done it via TEST_EXTRA_STACK_SIZE.